nautobot/extras/migrations/0084_taggeditem_cleanup.py

```diff
@@ -0,0 +1,43 @@
+from django.db import migrations, models
+
+
+def cleanup_taggeditem_duplicates(apps, schema_editor):
+    TaggedItem = apps.get_model("extras", "TaggedItem")
+
+    # Get the set of distinct values for TaggedItems
+    taggeditem_values = TaggedItem.objects.values("content_type", "object_id", "tag")
+    # Discard any existing ordering, just to be safe
+    taggeditem_values = taggeditem_values.order_by()
+    # Count the number of duplicates for each value
+    taggeditem_values = taggeditem_values.annotate(count_id=models.Count("id"))
+
+    # Filter to only those that actually have duplicates (i.e. more than one record)
+    duplicate_taggeditem_values = taggeditem_values.filter(count_id__gt=1)
```

Comment:
Maybe?
```python
    duplicate_taggeditems = (
        TaggedItem.objects.values("content_type", "object_id", "tag")
        .annotate(count_id=Count("id"))
        .filter(count_id__gt=1)
    )
```
And also what are we trying to be safe from by discard any existing ordering?